lib/ansible/modules/cloud/cloudstack/cs_ip_address.py

```diff
@@ -34,9 +34,11 @@
   network:
     description:
       - Network the IP address is related to.
+      - Mutually exclusive with C(vpc).
   vpc:
     description:
       - VPC the IP address is related to.
+      - Mutually exclusive with C(network).
```

Comment:
Should not it be written I(network) ans I(vpc) as it's option names?
I'm a little lost with these formats.

Reply:
@dpassante indeed, historically I used to use C() but I'll change that, thanks! https://docs.ansible.com/ansible/2.3/dev_guide/developing_modules_documenting.html#formatting-options